src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs

```diff
@@ -104,6 +104,20 @@ private void AddProject(SlnFile slnFile, string projectPath)
                 };
 
                 slnFile.Projects.Add(slnProject);
+
+                if (slnFile.ProjectConfigurationsSection.Any())
+                {
+                    var existingBuildConfigs = slnFile.ProjectConfigurationsSection.First();
```

Comment:
Could there be more than one and would that be considered valid? This would silently skip subsequent ones. `Single` would at least throw.